lib/rules/indent.js

```diff
@@ -113,6 +113,22 @@ module.exports = {
                                 minimum: 0
                             }
                         }
+                    },
+                    CallExpression: {
+                        type: "object",
+                        properties: {
+                            parameters: {
+                                oneOf: [
+                                    {
+                                        type: "integer",
+                                        minimum: 0
+                                    },
+                                    {
+                                        enum: ["first"]
```

Comment:
I see your point about this, but if we're going to explicitly add `"off"` values to options that are disabled by default, I think it should be part of a separate PR, since at the moment there are a lot of other `indent` options that don't have `"off"`.